File: src/priv/bajdcc/LALR1/syntax/automata/npa/NPAEdge.java
package priv.bajdcc.LALR1.syntax.automata.npa;

/**
 * 非确定性下推自动机边
 * 
 * @author bajdcc
 *
 */
public class NPAEdge {
	/**
	 * 初态
	 */
	public NPAStatus begin;

	/**
	 * 终态
	 */
	public NPAStatus end;

	/**
	 * 数据
	 */
	public NPAEdgeData data = new NPAEdgeData();
}

File: src/main/java/com/bajdcc/LALR1/syntax/automata/npa/NPAEdge.kt
package com.bajdcc.LALR1.syntax.automata.npa

/**
 * 非确定性下推自动机边
 * [begin] 初态
 * [end] 终态
 * [data] 数据
 * @author bajdcc
 */
data class NPAEdge (var begin: NPAStatus? = null, var end: NPAStatus? = null, var data: NPAEdgeData = NPAEdgeData())